lcUILua/createActions/createOperations.lua

```diff
@@ -86,6 +86,8 @@ function CreateOperations:close()
         luaInterface:triggerEvent('operationFinished', self.target_widget)
         self:removeTempEntity()
         self:unregisterEvents()
+        cli_get_text(0, false)
```

Comment:
The first argument of `cli_get_text` is the ID of the window, here, a second LibreCAD window would change this parameter on the other window.
You can access the current window ID in `self.target_widget`